aws_helper/config.go

```diff
@@ -38,6 +38,26 @@ func AssumeRoleEnvironmentVariables(roleArn string, sessionName string) (map[str
 	}
 
 	svc := sts.New(session)
+
+	if roleArn == "" {
+		// If no role is specified, we just convert the current access to environment variables
+		// if a role is assumed. This is required because terraform does not support AWS_PROFILE
+		// that refers to a configuration that assume a role.
+		cred, err := svc.Config.Credentials.Get()
+		if err != nil {
+			return nil, err
+		}
+
+		if cred.SessionToken == "" {
+			return nil, nil
+		}
+		return map[string]string{
```

Comment:
Personally, I prefer having only one `return` statement when I return the same object with only some different values in it. Instead of a `if` with a `return` statement in it, it could be a `if else` with a common `return` at the end.